run/cifs_archive/archive-clips.sh

```diff
@@ -37,6 +41,13 @@ function moveclips() {
         then
           log "Moved '$file_name'"
           NUM_FILES_MOVED=$((NUM_FILES_MOVED + 1))
+
+          # Every 10 files, send a wakeup command to the car via the Tesla API,
```

Comment:
why every 10 files? How long does the tesla api wakeup command keep the car awake?

Reply:
I was curious about this. I just ran a shell script to quickly check this. It looks like about 10 minutes before it is asleep. offline does not cut power. 
```
Tesla is asleep at Mon  3 Jun 23:19:52 CDT 2019
Tesla is asleep at Mon  3 Jun 23:20:53 CDT 2019
Tesla is online at Mon  3 Jun 23:21:54 CDT 2019
Tesla is online at Mon  3 Jun 23:22:55 CDT 2019
Tesla is online at Mon  3 Jun 23:23:56 CDT 2019
Tesla is online at Mon  3 Jun 23:24:56 CDT 2019
Tesla is online at Mon  3 Jun 23:25:58 CDT 2019
Tesla is online at Mon  3 Jun 23:26:58 CDT 2019
Tesla is offline at Mon  3 Jun 23:27:59 CDT 2019
Tesla is offline at Mon  3 Jun 23:29:00 CDT 2019
Tesla is offline at Mon  3 Jun 23:30:00 CDT 2019
Tesla is offline at Mon  3 Jun 23:31:01 CDT 2019
Tesla is online at Mon  3 Jun 23:32:02 CDT 2019
Tesla is asleep at Mon  3 Jun 23:33:03 CDT 2019
Tesla is asleep at Mon  3 Jun 23:34:04 CDT 2019
```
I think a more robust way would be to set it to wake every 8 minutes. Use the built-in bash SECONDS variable to check this - https://stackoverflow.com/questions/8903239/how-to-calculate-time-difference-in-bash-script

Reply:
I did it every ten files because that seemed fairly reasonable and easy to program, and would definitely be fast enough under the speed at which my own Pi archives to my PC. As mentioned in other comments, I am a total bash programming amateur, so I had no knowledge of any sort of alternative timing mechanism besides simply inserting the call between X number of file moves.

I'll see if I can figure out how to use the `SECONDS` variable, though I think 10 minutes may be too long. The whole point is to prevent the car from going offline in the first place.